schema/schema.go

```diff
@@ -483,6 +483,19 @@ func (schema *Schema) StateVersioning() bool {
 	return stateful
 }
 
+//Disable like search wheter to use substring search or full string search on column values
+func (schema *Schema) SearchLikeDisabled() bool {
+	likeDisabled, ok := schema.Metadata["disable_substr_search"]
```

Comment:
since the underlying default behavior of search is changing ..enabling it rather than disabling through a flag means we would have to alter it in all the schema files..whereas based on our discussion we just need to disable a handful of schema's..so disabling it in a few is much simpler..

Reply:
good point. nevertheless, I wouldn't mind changing the method name (cf. comment above)